tests/hw-accel/kmm/internal/define/module.go

```
@@ -72,3 +101,79 @@ func DRASpec(serviceAccountName string, deviceClassNames []string,
+func DRASpecWithCELSelector(serviceAccountName string, deviceClassNames []string,
+	extraEnv []map[string]interface{}) map[string]interface{} {
+	spec := DRASpec(serviceAccountName, deviceClassNames, extraEnv)
+
+	classes, ok := spec["deviceClasses"].([]interface{})
+	if !ok {
+		return spec
+	}
+
+	expression := fmt.Sprintf("device.driver == '%s'", kmmparams.DRADriverName)
+
+	for i, class := range classes {
```

golangci-lint (2.13.2)

[error] variable name 'i' is too short for the scope of its usage

(varnamelen)